test/integration/scheduler/plugins/plugins_test.go

```diff
@@ -216,13 +328,16 @@ func (sp *ScorePlugin) Name() string {
 
 // Score returns the score of scheduling a pod on a specific node.
 func (sp *ScorePlugin) Score(ctx context.Context, state *framework.CycleState, p *v1.Pod, nodeName string) (int64, *framework.Status) {
-	curCalled := atomic.AddInt32(&sp.numScoreCalled, 1)
+	sp.mutex.Lock()
```

Comment:
But now all score calls are essentially serialized.

Arguably, we are calling these plugins with only a few nodes. But this is not how plugins work in production.

Reply:
Do you really think that mutex vs. atomic.AddInt32 makes any difference? The lock isn't going to be held long.

Reply:
Also remember that this is a test for correct behavior, not performance. It shouldn't depend on timing.

Reply:
Add a comment in the plugins to clarify that they shouldn't be used for benchmarks, because they hold a lock.

Reply:
Instead of repeating that comment for each individual plugin, I added a package comment:
https://github.com/kubernetes/kubernetes/compare/9f556132010572c93f757a0f71b086d3743738cb..9e9a6cde4b8b446cf68704ceb6e52df7eb1e00e3